pages/facts.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fun Facts about Succulents</title>
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/facts.css">
+  </head>
+  <body class="container">
+    <header id="blue">
+      <h1>The Succulent Spot</h1>
+      <nav id="navlinks">
+        <ul class="nav-container">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">Facts</a></li>
+          <li><a href="./gallery.html">Gallery</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h2>3 Fun Facts About Succulents</h2>
+        <ul class="facts-container">      
+          <li class="fact">
+            <span class="fact-text">The word "succulent" comes from the Latin word <i>sucus</i>, meaning "juice" or "sap".</span>
+          </li>
+          <li class="fact">
+            <span class="fact-text">The largest succulent in the world is the baobab tree. The main stem of the baobab may reach up to 92 feet in girth and 82 feet in height. With its distinctive silhouette (think Rafiki's tree in <i>The Lion King</i>), the baobab is perhaps the best known of all African trees, and it is known by the local people as the "tree of life".</span>
+          </li>
+          <li class="fact">
+            <span class="fact-text">I show my fluffy belly but it's a trap! if you pet it i will tear up your hand ask to be pet then attack owners hand yet hiding behind the couch until lured out by a feathery toy and claws in the eye of the beholder catasstrophe and bleghbleghvomit my furball really tie the room together.</span>
+          </li>
+          <li class="fact">
+            <span class="fact-text">Some succulents produce their own "sunscreen". Unlike human sunscreen, however, succulents use a waxy, almost dust-like coating called epicuticular wax to protect leaves from burning if there is too much light.</span>
```

Comment:
Consider using `<p>` for this text since it is several sentences. `span` is more for highlighting a few words or a sentence that need to be styled differently within the paragraph.